acceptance/tests/30_install_test_remote_master_tarball_path_master.rb

```diff
@@ -0,0 +1,36 @@
+require "./acceptance/helpers/beaker_helper"
+
+test_name "perform install on remote master with tarball path on master" do
+  step "perform install" do
+    filename = BeakerHelper.get_pe_tarball_filename(target_master)
+    primary_master = "--primary-master=#{target_master}"
+    pe_tarball = "--pe-tarball=#{target_master}:/tmp/ras/#{filename}"
+    pe_conf = "--pe-conf=/root/ref_arch_setup/fixtures/pe.conf"
+    command = "ref_arch_setup install #{primary_master} #{pe_tarball} #{pe_conf}"
+
+    puts command
+    on controller, command
+  end
+
+  step "run puppet agent" do
+    on target_master, "puppet agent -t"
+  end
+
+  teardown do
+    # TODO: make this a task
```

Comment:
Move this teardown functionality to the beaker helper, and then call it from each test it is used in.

Reply:
Updated